codechecker_lib/analyzer.py

```diff
@@ -134,6 +131,9 @@ def run_check(args, actions, context):
     end_time = time.time()
 
     with client.get_connection() as connection:
+        if os.path.exists(suppress_file):
+            client.send_suppress(context.run_id, connection, suppress_file)
+
```

Comment:
Is there any reason you moved this? I see no problem with the moving itself, but was wondering whether this is a fix or just a refactoring with no functional change.